src/cell_type_mapper/diff_exp/score_utils.py

```diff
@@ -317,3 +319,24 @@ def aggregate_stats(
             result[k] = result[k].astype(new_dtype)
 
     return result
+
+
+def _read_cluster_to_row(cluster_row_lookup):
```

Comment:
Please add a unit test for this function. I know it's trivial, I just want to make sure that this is actually solving the problem we want it to solve.